src/use_case/vote_out/VoteOutInteractor.java

```diff
@@ -1,4 +1,39 @@
 package use_case.vote_out;
 
+import entity.Game;
+
 public class VoteOutInteractor implements VoteOutInputBoundary {
+    final VoteOutDataAccessInterface userDataAccessObject;
+    final VoteOutOutputBoundary userPresenter;
+    final Game game;
+
+    public VoteOutInteractor(VoteOutDataAccessInterface userDataAccessObject, VoteOutOutputBoundary userPresenter, Game game) {
+        this.userDataAccessObject = userDataAccessObject;
+        this.userPresenter = userPresenter;
+        this.game = game;
+    }
+
+    // This method returns true if a werewolf is voted out and returns false otherwise
+    // Thus, it is false if a villager is voted out OR if an invalid name is entered
+    @Override
+    public boolean voteOutPlayer(VoteOutInputData voteOutInputData) {
```

Comment:
Could you comment in the code why we need to return a boolean?

The main reason I do not understand why a boolean is returned is because I thought we just execute prepareFailView or prepareSuccessView.

Additionally, where does the story get generated? That should be part of the output data since someone is getting killed.

Reply:
okay yeah ill make it a void then the success/fail views are fine for our purposes. 
though I'm pretty sure my use case shouldn't have to worry about generating stories at all?